extensions/oidc-common/deployment/src/main/java/io/quarkus/oidc/common/deployment/devservices/keycloak/KeycloakDevServicesProcessor.java

```diff
@@ -1,4 +1,7 @@
-package io.quarkus.oidc.deployment.devservices.keycloak;
+package io.quarkus.oidc.common.deployment.devservices.keycloak;
```

Comment:
I was thinking it would actually end up in the devservices set of extensions, here, https://github.com/quarkusio/quarkus/tree/main/extensions/devservices (so far it hosts database specific services only but we can add `oidc` there, may be in the `security` subfolder), so that `keycloak-admin-client` can eventually pick it up too.

Reply:
Well, I don't mind. I am careful when creating new modules, I don't create it until there is requirement, but if that's what you want, I'll just change it.

Reply:
Thanks @michalvavrik, IMHO it would be the next requirement from the Keycloak-admin-client users anyway :-), like, if you can do it with OIDC client, why can't we do it with keycloak-admin-client :-). I'm pretty sure we had some related queries

Reply:
I have separated it (still testing locally) I just want to clarify I won't provide Dev Svc for KC Admin client ATM, it will give us opportunity to implement it with one trivial PR, but we need to  keep this scope because it has potential for merge conflicts or on missing something that will change between now and merging.